neural_network/2_hidden_layers_neural_network.py

```diff
@@ -0,0 +1,203 @@
+import numpy
+
+
+class NeuralNetwork:
```

Comment:
Please provide a better class name here. What type of neural network is this class representing?